.gitignore

```diff
@@ -53,3 +53,6 @@ Makefile.in
 
 .deps
 .dirstamp
+
+# OS X
+.DS_Store
```

Comment:
What's this?

Reply:
I'm developing on macOS. This is kind of standard file generated almost in each macOS folder, something like desktop.ini in Microsoft Windows. https://en.wikipedia.org/wiki/.DS_Store
It's normal to add it to gitignore